docs/faststore/docs/go-live/1-configuring-external-dns.mdx

```diff
@@ -1,155 +1,113 @@
 ---
 title: '1. Configuring external DNS'
-excerpt: 'Learn how to point your custom domain to FastStore and get your store ready for production.'
-createdAt: '2024-01-01T00:00:00Z'
-updatedAt: '2025-09-23T00:00:00Z'
 ---
 
-In this guide, you will learn how to point your custom domain to your **FastStore** website and configure it in VTEX for a complete setup. The process involves the following steps:
+In this guide, you will learn how to configure external domains for your **FastStore** and **VTEX** website. The process involves the following steps:
```

Comment:
```suggestion
In this guide, you will configure your custom domain so it correctly points to your FastStore website hosted on [FastStore WebOps](https://developers.vtex.com/docs/guides/faststore/webops-dashboard) (`https://{accountName}.vtex.app/`).
Once the configuration is complete, customers will access your store using your final domain (e.g., `www.mystore.com`) instead of the default WebOps URL.
```